### PR TITLE
Admin create pub

### PR DESCRIPTION
**Pull request by Sam-Archie**

### CODE REVIEW

**Comment by ruarim** [comment]
Is this branch just for beverages? How come its called pub attributes?

**Comment by Sam-Archie** [comment]
@RuariM ignore branch name it has nothing to do with that, might have been my original intention but is not what it has become.

File: database/migrations/2022_11_01_163211_create_beverages_table.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            $table->string('brewer');
+            $table->string('abv');
+            $table->string('type');
+            $table->string('country')->nullable();
+            $table->string('region')->nullable();
+            $table->string('style')->nullable();
+            $table->string('producer')->nullable();
+        });
+
+        Schema::create('venue_beverage', function (Blueprint $table) { // still have an id column
```
Comment (ruarim) [inline database/migrations/2022_11_01_163211_create_beverages_table.php]: Iv been having issues when I call my link table venue_attribute but attribute_venue works.

So I wonder if this should be called beverage_venue?

Comment (Sam-Archie) [reply database/migrations/2022_11_01_163211_create_beverages_table.php]: hmm interesting ill have a better look around this in the evening.

**Pull request merged**